fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -836,6 +839,11 @@ showProcStatement
     : SHOW PROC path=string
     ;
 
+// ---------------------------------------- Backup Store Statement -----------------------------------------------------
+showRestoreStatement
+    : SHOW RESTORE (FROM identifier)?
```

Comment:
miss where clause?

Reply:
I see there is no where clause in the official document.https://docs.starrocks.com/zh-cn/main/sql-reference/sql-statements/data-manipulation/SHOW%20RESTORE

Reply:
The documentation is not comprehensive and you should follow the source code， you can test the where clause using old parser and analyzer

Reply:
It has been modified